tokio/src/runtime/task/list.rs

```diff
@@ -170,11 +171,7 @@ impl<S: 'static> OwnedTasks<S> {
     }
 
     pub(crate) fn remove(&self, task: &Task<S>) -> Option<Task<S>> {
-        let task_id = task.header().get_owner_id();
-        if task_id == 0 {
-            // The task is unowned.
-            return None;
-        }
+        let task_id = task.header().get_owner_id()?;
```

Comment:
I'd like at least a comment here to point out what's going on.

Reply:
Done.